cn_indian_payroll/cn_indian_payroll/overrides/declaration.py

```
@@ -420,36 +198,18 @@ def choose_regime(doc_id, employee,payroll_period,company,regime):
 
                 num_months = (end.year - start.year) * 12 + (end.month - start.month)+1
 
-                            
+                total_pt=round(num_months*pt_amount)          
```

Ruff (0.8.2)

Local variable `total_pt` is assigned to but never used

Remove assignment to unused variable `total_pt`

(F841)